### ISSUE TITLE
Support more languages for syntax highlighting in code blocks

### ISSUE THREAD

**Issue by somelinguist** [MarkEdit-app/MarkEdit#14]
Hi. What are your thoughts on supporting more languages for syntax highlighting in code blocks?

I have a fork (https://github.com/somelinguist/MarkEdit) that enables highlighting for fsharp and ocaml using `@codemirror/legacy-modes/mode/mllike`, which is already included in the app but not enabled.

Would you be open to me submitting a pull request?

Thanks for this great app!

**Comment by somelinguist** [comment]
Hi @cyanzhong. Thanks for sharing your thoughts about it. It makes sense to want to keep the app small and agile. 

I've opened the pull request here https://github.com/MarkEdit-app/MarkEdit/pull/15

Is there anything I missed? It built everything on my machine and ran the tests with everything passing, and it works in the all. But I don't know if anything else is needed to make it complete.

F# is one of the languages I use frequently these days, so it was one of the first things I checked out in MarkEdit. :) I was glad to see that CodeMirror already had support for it.

Thanks again for the app.
